app/service-cli-bytearray.cpp

```diff
@@ -28,10 +28,11 @@ int main(int argc, char *argv[]) {
   std::future<Response> responseFuture = service.translate(std::move(input));
   responseFuture.wait();
   Response response = responseFuture.get();
-  std::cout << response.translation() << std::endl;
+  std::cout << response.target.blob << std::endl;
```

Comment:
This is a downgrade to the API.

Reply:
I'm not sure I understand what to do here, do I rename this back and expose the old method .. `translation`()? Or unbind the blob + annotation structure?

Reply:
This is a struct. `response.target.text` works? This is intended to map to what we call a "blob of text".

Reply:
Renamed to text. AnnotatedBlob is now AnnotatedText.